flavor.json

```diff
@@ -50,57 +50,76 @@
         {
           "seconds": 4,
           "text": [
-            "Welcome to the game",
+            "Velcom all, and thanks vor coming to my Vampire Party",
             "Salutations and welcome to Vampire Party",
-            "Thanks for joining the party"
+            "Thanks for joining the party",
+            "So glad you could make it. You won't live....to regret it"
           ]
         },
         {
-          "seconds": 3,
-          "text": "This is some text"
+          "seconds": 9,
+          "text": "Here in our gloomy town of Spoons we seem to have run into a bit of a vampire problem. Maybe because of our unforunate positioning ontop of a gaping..HellNose. Who is to really know?"
         },
         {
-          "seconds": 3,
-          "text": "with a variety of timers"
+          "seconds": 6,
+          "text":
+            "What I do know though, is that there are vampires and people are dying, and that just makes me sad?"
+        },
+          {
+          "seconds": 10,
+          "text": "Wait, hold on. Is your videochat and microphone turned on? And is {vipPlayer} sharing thier screen? If not, pause the game and do that now, thanks <3 xoxo"
         },
         {
-          "seconds": 4,
-          "text": "once the last timers runs out..."
+          "seconds": 15,
+          "text": "Ok great. So here's the deal. I'm gonna need y'all to figure out who these dang vampires are. They seem to always attack at night. So I figured, during the day we can all meet up around 3 PM to perform my favorite vampire staking ceremony? Yay! So Fun! And then, we can just keep doing this everyday until we kill all the vampires, or until they overrun the town. Yeah? Awesome! " 
+        },
+        {
+          "seconds": 10,
+          "text": "Majority rules is the rule in Spoons. That goes for the Vampires too. When it comes time to Stake (or Suck, no judgement), and you cannot come to an agreement, no one will die."
```

Comment:
"That goes for the Vampires too". Does it?  I can update rules, but currently I have it setup so they all have to agree.
It's fine either way, but something to be noted so we're on the same page.